src/Data/Auth/Providers/JwtProvider.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { useContext, useEffect, useState } from "react";
+import { useGetCurrentUser } from "@/Data/Managers/V2/GetCurrentUser";
+import { AuthContext } from "../AuthContext";
+/**
+ * Component that implements a null authentication provider when no authentication is enabled.
+ */
+export const JwtProvider: React.FC<React.PropsWithChildren> = ({
+  children,
+}) => {
+  const [user, setUser] = useState<string | null>(null);
+  const { data, isSuccess } = useGetCurrentUser().useOneTime();
+  const authContext = useContext(AuthContext);
+
+  const getUser = (): string | null => user;
+  const isDisabled = () => true;
+
+  useEffect(() => {
+    if (isSuccess) {
+      setUser(data?.username);
+    }
+  }, [data, isSuccess]);
+  return (
```

Comment:
new line before return
```suggestion
  
  return (
```